JSDemos/Demos/TabPanel/SortableClosableTabs/React/App.js

```diff
@@ -36,40 +35,41 @@ function App() {
     if (index >= newEmployees.length && index > 0) {
       setSelectedItem(newEmployees[index - 1]);
     }
-  }
+  }, [employees, setEmployees, setSelectedItem]);
 
-  function renderTitle(data) {
+  const renderTitle = React.useCallback((data) => {
     function closeHandler() {
       closeButtonHandler(data);
     }
+
     return (
       <React.Fragment>
         <div>
           <span>
             {data.FirstName} {data.LastName}
           </span>
-          {employees.length >= 2 && <i className="dx-icon dx-icon-close" onClick={closeHandler} />}
+          {employees.length >= 2 && <i className="dx-icon dx-icon-close" onClick={() => { closeHandler(); }} />}
         </div>
       </React.Fragment>
     );
-  }
+  }, [closeButtonHandler]);
```

Comment:
shouldn't `employees` be in deps?

Reply:
You're right! Great job! https://github.com/DevExpress/devextreme-demos/pull/1916/commits/15f8186a1bee53602904eb8267b2fcd2f0b6b1f2